libsplinter/src/biome/user/store/diesel/operations/add_user.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Cargill Incorporated
```

Comment:
2020

Reply:
This was intentional. I'm going to separately propose that we replace all the copyright header lines at the same time using the same (2019-2020) copyright.